cloud_governance/tag_non_cluster/run_tag_non_cluster_resources.py

```diff
@@ -50,7 +50,7 @@ def tag_na_resources(file_name: str, file_path: str = '/tmp/', tag_operation: st
     @param file_name:
     @return:
     """
-    file_name = f'{file_path}{region}-{file_name}'
+    file_name = f'{file_path}{file_name}'
```

Comment:
Why did u remove the region ?

Reply:
It causes noise. When giving filename.
Ex: the generated filename is us-west-1-tag_user.csv
given filename is us-west-1-tag_user.csv. It added us-west-1-us-west-1-tag_user.csv. So I removed it.